examples/renameimage.py

```diff
@@ -28,6 +28,65 @@
 
 from glancesync.glancesync import GlanceSync
 
+"""This code renames an image in all the regions. The image must exist in
+the master region (it is OK if the image has been already renamed). If the
+image does not exists in the remote regions, it is silently ignored.
+
+The image is only renamed if the checksum is the same than the image in the
+master region. Also the is_public property is updated. This code is also a
+way to deprecated an image: adding the suffix "deprecated" and changing
+is_public to False.
+
+If there is more than image in a region matching the name and checksum, an
+error is printed and no image is renamed. If a image to rename is found but a
+image with the name of the new image already exists, an error is printed and
+the image is not renamed. Only the errors in the master region halts the
+program.
+
+The properties in sync_properties are also synchronised when the image is
+updated.
+"""
+
+sync_properties = ['sdc_aware']
+
+def update_image(image, image_master):
+    """
+    Update the image with the name and contents of image_master
+    :param image:
+    :param image_master:
+    :return: nothing
+    """
+    image.name = image_master.name
+    image.is_public = image_master.is_public
+    for prop in sync_properties:
+        if prop in image_master.user_properties:
+            image.user_properties[prop] = image_master.user_properties[prop]
+
+    glancesync.update_metadata_image(image.region, image)
+
+def metadata_outdated(image, image_master):
+    """Compare metadata of image and image_master; return
+    True if image needs to be updated
+
+    :param image: the image in the region
+    :param image_master:  the master image
+    :return: True if the image needs an update, False otherwise.
+    """
+    if image.is_public != image_master.is_public:
```

Comment:
I do not like to have more than one return in the same method, try to rewrite the code.